Fabric.Authorization.AccessControl/src/app/modules/access-control/grain-list/grain-list.component.spec.ts

```diff
@@ -72,7 +72,7 @@ describe('GrainListComponent', () => {
 
     // assert
     expect(component.selectedNode).not.toEqual(null);
-    expect(component.selectedNode.name).toBe(grains[0].name);
+    expect(component.selectedNode.name).toBe(grains[0].securableItems[0].name);
```

Comment:
The grains-list will default to the first securable item when none is explicitly passed in